database/factories/InsegnamentoFactory.php

```diff
@@ -24,7 +24,7 @@
         'docente'           => $faker->lastName . ' ' . $faker->firstName,
         'canale'            => array('AL', 'MZ', null)[rand() % 3],
         'id_modulo'         => $idModulo,
-        'nome_modulo'       => $nomeModulo, 
```

Comment:
Nulla di modificato in questo file, come mai è contenuto nel commit?

Reply:
lo avro' aperto e salvato, un plugin di vscode avra' rimosso gli spazi inutili